kube-scheduler-healthcheck

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -eu
+TEST_POD_NAMESPACE="${TEST_POD_NAMESPACE:-kube-system}"
+TEST_POD_IMAGE_LOCATION="${TEST_POD_IMAGE_LOCATION:-gcr.io/google_containers}"
```

Comment:
Move `google_containers` to the next line

Reply:
@stuart-warren recommended getting rid of the split all together. Which way are we going?